core/src/components/modal/modal.tsx

```diff
@@ -833,7 +893,16 @@ export class Modal implements ComponentInterface, OverlayInterface {
         {mode === 'ios' && <div class="modal-shadow"></div>}
 
         <div role="dialog" class="modal-wrapper ion-overlay-wrapper" part="content" ref={(el) => (this.wrapperEl = el)}>
-          {showHandle && <div class="modal-handle" part="handle"></div>}
+          {showHandle && (
+            <button
+              class="modal-handle"
+              // Prevents the handle from receiving keyboard focus when it does not cycle
+              tabIndex={!isHandleCycle ? -1 : 0}
+              aria-label={isHandleCycle ? 'Activate to adjust the size of the dialog overlaying the screen' : null}
```

Comment:
Even when handleCycle is `none`, the handle can still be used to adjust the size of the dialog. Should we keep the aria-label here? Or maybe have a different aria label?